src/diffusers/modeling_utils.py

```diff
@@ -666,20 +666,6 @@ def num_parameters(self, only_trainable: bool = False, exclude_embeddings: bool
             return sum(p.numel() for p in self.parameters() if p.requires_grad or not only_trainable)
 
 
-def unwrap_model(model: torch.nn.Module) -> torch.nn.Module:
-    """
-    Recursively unwraps a model from potential containers (as used in distributed training).
```

Comment:
Not technically deprecated, but we've had this replaced by `accelerate.unwrap_model` a loooong time ago.

Reply:
Makes sense!